guardian/automat
# Go API revisited

**TODO: add ES store implementation.**

## What does this change?

A Go(lang) implementation of the API. Originally we expected a wider team and cross-team involvement even early on and discarded Go on that basis. But as that is no longer the case, the productivity benefits of Go become relevant again.

## How to test

    // in api-go/
    gin run main.go

    // in ui/
    export REACT_APP_AUTOMAT_API_URL='http://localhost:3000'
    yarn start

And things should just work. (It uses an in-memory db for now.)
